grails-app/conf/Config.groovy

```diff
@@ -49,12 +49,13 @@ environments {
     }
 }
 
+// production uses Chef managed values
```

Comment:
@pmbohm You commented in the wrong place - the discussion is here.

Again, this is implied, it doesn't matter how much time you and Loz spent. If you think those defaults should be changed - then change them (on a different commit of course). This repository has nothing to do with chef.